webpack.prod.js

```diff
@@ -0,0 +1,40 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+
+module.exports = merge(common, {
+    mode: 'production',
+    devtool: 'cheap-source-map',
+    optimization: {
```

Comment:
what about having another output here like:
```
output: {
        filename: '[name].[contenthash].js',
        path: path.join(__dirname, 'dist'),
    },
```

so it is going to split the bundle into vendor and main.